packages/buckets/src/buckets.spec.ts

```diff
@@ -26,15 +26,21 @@ describe('Buckets...', () => {
 
   before(async () => {
     const user = await signUp(ctx, addrGatewayUrl, sessionSecret)
-    const id = ThreadID.fromRandom()
-    const db = new Client(ctx.withSession(user.user?.session).withThreadName('buckets'))
-    await db.newDB(id)
+    ctx.withSession(user.user?.session)
   })
 
-  it('should init a new bucket', async () => {
+  it('should open a bucket by name without thread info', async () => {
+    const root = await client.open('initbuck')
+    expect(root).to.have.ownProperty('key')
+    expect(root).to.have.ownProperty('path')
+    expect(root).to.have.ownProperty('createdat')
+    expect(root).to.have.ownProperty('updatedat')
+  })
+
+  it('should init a new bucket on open thread', async () => {
     // Check that we're empty
     const list = await client.list()
-    expect(list).to.have.length(0)
+    expect(list).to.have.length(1)
     // Now initialize a bucket
     buck = await client.init('mybuck')
```

Comment:
init could just go away, but it still works fine the same way as before too. it's just complicated